vue.config.js

```diff
@@ -107,17 +107,17 @@ if (!isProd) {
 }
 
 // Define npm module resolve order: 1. local (Twill), 2. root (App)
-const appModuleFolder = path.resolve(__dirname, '../../../node_modules') // vendor/area17/twill/
-const resolveModules = ['node_modules']
-if (fs.existsSync(appModuleFolder)) {
-  resolveModules.push(appModuleFolder)
-}
+// const appModuleFolder = path.resolve(__dirname, '../../../node_modules') // vendor/area17/twill/
```

Comment:
@ifox No side effect on the changes I did in this file afaik. But maybe they are for something special I am not aware of.